src/test/java/com/acme/conferencesystem/ContainerConfig.java

```diff
@@ -13,7 +13,8 @@ public class ContainerConfig {
     @ServiceConnection
     @RestartScope
     public PostgreSQLContainer<?> postgreSQLContainer() {
-        return new PostgreSQLContainer<>("postgres:16-alpine");
+        return new PostgreSQLContainer<>("postgres:16-alpine")
+                .withCommand("postgres", "-c", "max_connections=200");
```

Comment:
what for?

Reply:
There is an issue with the connections, I think with flyway. I don't have capacity right now to have a look, so it was a work arround xD